src/Sql/Sql.Test/ScenarioTests/AuditTests.cs

```diff
@@ -301,12 +301,14 @@ public void TestServerAuditingToStorageInVNet()
             TestRunner.RunTestScript("Test-ServerAuditingToStorageInVNet");
         }
 
+/*  TODO need to fix test json.
```

Comment:
Please refer to https://github.com/Azure/azure-powershell/blob/5d2a6013b6a4c97ea59a2165d01b3981a19d4aab/src/Sql/Sql.Test/ScenarioTests/ManagedInstanceProtectorTests.cs#L30 and skip the test